src/containers/viewSingleRequest/index.tsx

```diff
@@ -0,0 +1,185 @@
+import { useParams } from 'react-router-dom';
+import React, { useEffect, useState } from 'react';
+import {
+  AsyncRequest,
+  AsyncRequestCompleted,
+  AsyncRequestFailed,
+  asyncRequestIsComplete,
+  asyncRequestIsFailed,
+  asyncRequestIsLoading,
+  asyncRequestIsNotStarted,
+  AsyncRequestLoading,
+  AsyncRequestNotStarted,
+} from '../../utils/asyncRequest';
+import { ContactInfo } from '../setContacts/ducks/types';
+import { Helmet } from 'react-helmet';
+import { ChungusContentContainer, ContentContainer } from '../../components';
+import ContactInfoSummary from '../../components/ContactInfoSummary';
+import { Alert, Spin, Button } from 'antd';
+import styled from 'styled-components';
+import { LinkButton } from '../../components/LinkButton';
+import protectedApiClient from '../../api/protectedApiClient';
+import DecisionConfirmation from '../../components/DecisionConfirmation';
+
+enum Status {
+  PENDING,
+  APPROVED,
+  DENIED,
+}
+
+const DecisionButtons = styled.div`
+  display: flex;
+  justify-content: center;
+  align-items: center;
+`;
+
+const StyledButton = styled(Button)`
+  margin: 5px;
+`;
+
+const StyledAlert = styled(Alert)`
+  margin-left: 20%;
+  max-width: 60%;
+`;
+
+const ViewSingleRequest = () => {
+  const requestId = Number(
+    useParams<{ request_id: string; user_id: string }>().request_id,
+  );
+  const userId = Number(
+    useParams<{ request_id: string; user_id: string }>().user_id,
+  );
+  const [contacts, setContacts] = useState<AsyncRequest<ContactInfo, any>>(
+    AsyncRequestNotStarted(),
+  );
```

Comment:
Isn't there a contacts store? Why can't we just select this data from the store and dispatch the thunk to load if it's not present? Then we don't have to duplicate this code